core/primitives/Cargo.toml

```diff
@@ -18,6 +18,7 @@ byteorder = "1.3"
 chrono = { version = "0.4.4", features = ["serde"] }
 derive_more = "0.99.3"
 easy-ext = "0.2"
+include_dir = "0.6.1"
```

Comment:
Is it possible to avoid this? Ideally we don't want to introduce new dependencies if possible